docs/products/calloption.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Call Options
+sidebar_label: Call Options
+---
+A call option is a building block for the derivatives market. It gives the buyer the right but not the obligation to buy the asset at a specific price (strike) at a specific time. 
+
+This page will detail how to create your own call options and layout a few reasons why you would want to create call options.
+
+## How options work
+
+A trader buys a call option with a strike price of 2,000 USD for 0.05 ETH. Now he has the right to buy 1 ETH for 2,000 USD. At the expiry, ETH/USD is at 2,500 USD, and the delivery price is 2,500 USD.
+
+In this case, the option is settled for 500 USD per 1 ETH. At the expiry, the trader’s account is credited with 0.2 ETH (500/2,500), and the seller’s account is debited with 0.2 ETH. The initial purchase price was 0.05 ETH; therefore, the trader’s profit is 0.15 ETH.
+Any call option with an exercise price (strike price) above 2,500 USD will expire worthless.
```

Comment:
Having this sentence here feels a little confusing; I would just cut it. The next sentence, explaining how the specific call option bought by the trader could have hypothetically expired worthless, is good.